### DISCUSSION TITLE
Another way to display cover and event flyer?

### DISCUSSION THREAD

**Discussion by islandmix** [HiEventsDev/Hi.Events#185] [General]
Currently, if someone uploads a portrait dimension event flyer for the cover page... then the image will show and the event details gest pushed down... i have seen other platforms crop portrait cover images... to keep the header consistent.

I am also confused with the concept of COVER PAGE and where I can load event flyer images... is the cover page supposed to be the event flyer? Most event flyers are double sided and sometimes more than 2 sides. 

Is there another way to show the event flyer image, cover image and avoid the scroll?

**Comment by daveearley** [comment]
The recommended dimensions for the cover image are 2160x1080px, so once to stick to a similar aspect ratio it should look OK.

The cover image isn't indented to be an event flyer, but I wouldn't be against allowing people to upload multiple images. If there's more requests for it I can add it to the roadmap. 

<img width="614" alt="Screenshot 2024-09-02 at 21 27 20" src="https://github.com/user-attachments/assets/cb364df1-9af3-4095-8e5c-3744d16ef7b6">
